modules/rosa-sts-creating-cluster.adoc

```diff
@@ -105,16 +106,24 @@ OIDC Endpoint URL:          https://rh-oidc.s3.us-east-1.amazonaws.com/<cluster_
 ====
 If installation fails or the `State` field does not change to `ready` after 40 minutes, check the installation troubleshooting documentation for more details.
 ====
-. Enter the following command to follow the OpenShift installer logs to track the progress of your cluster creation:
+. Set the environment variable for the `cluster_id` from the previous output:
```

Comment:
Consider this edit: Set the environment variable for the 'cluster_id'.

Reply:
My concern with not mentioning the previous output, is that it may be easy to miss this if they don't know where they're looking.